beebop/app.py

```diff
@@ -48,6 +54,31 @@ def check_connection(redis):
         abort(500, description="Redis not found")
 
 
+def generateZip(path_folder):
+    memory_file = BytesIO()
+    with zipfile.ZipFile(memory_file, 'w', zipfile.ZIP_DEFLATED) as zipf:
+        for root, dirs, files in os.walk(path_folder):
+            for file in files:
+                zipf.write(os.path.join(root, file), arcname=file)
+    memory_file.seek(0)
+    return memory_file
+
+
+def add_data(path, type, data):
+    with open(path, 'r') as file:
+        data[type] = file.read()
+
+
+def modify_microreact(json_microreact, project_hash, data):
```

Comment:
Can this be more informatively named? What are we modifying? (or add a comment/docstring)